server/models/articles.model.js

```diff
@@ -34,4 +34,14 @@ exports.fetchArticles = () => {
     `).then(({rows})=>{
          return rows
     })
+}
+
+exports.fetchComments = (article_id) => {
+    return db.query(`
```

Comment:
To deal with the case where we have no comments on a non-existent article, we'll need to do a bit of a check as to whether the article exists or not. If it does, we can assume it has no comments if the length of the result is zero. If it doesn't exist, we can treat is a 'not found'. I think looking at this I can see what you were trying to do in the controller, but that logic should really be here in the model.